main.rb

```diff
@@ -0,0 +1,15 @@
+# Import my_enumerable.rb
+
+require_relative 'my_enumerable'
+
+# Create MyList class
+class CustomList
+  include MyEnumerable
+  def initialize(*items)
+    @items = items
+  end
+
+  def iterate(&block)
+    @items.each { |item| block.call(item) }
+  end
+end
```

Comment:
- [ ] According to the project requirements, you should create a class `MyList` that has an instance variable @list. Kindly rename the CustomList class name to `MyList`  as requested by the project requirements. 👇🏻 

- [ ] Same here you should implement  In MyList class a method `#each` that yields successive members of @list and uses the MyEnumerable module. Kindly rename your iterate method name. 

![image](https://github.com/shayan1234554321/enumerable/assets/79058364/384e74e3-ae05-4d4f-a866-f7cbd1787d95)